packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type LinkExpiredProps = {
+  linkType: LinkType;
+};
+
+type LinkType = 'reset-password' | 'signin';
+
+const getResendLinkHandler = (linkType: LinkType) => {
+  if (linkType === 'reset-password') {
+    return function () {
+      //   TODO: add resend link action
+    };
+  }
+  return function () {
+    //   TODO: add resend link action
+  };
+};
+
+const getTemplateValues = (linkType: LinkType) => {
+  let templateValues = {
+    headerText: '',
+    headerId: '',
+    messageText: '',
+    messageId: '',
+  };
+  switch (linkType) {
+    case 'reset-password':
+      templateValues.headerText = 'Reset password link expired';
+      templateValues.headerId = 'reset-pwd-link-expired-header';
+      templateValues.messageText =
+        'The link you clicked to reset your password is expired.';
+      templateValues.messageId = 'reset-pwd-link-expired-message';
+
+      break;
+    case 'signin':
+      templateValues.headerText = 'Confirmation link expired';
+      templateValues.headerId = 'signin-link-expired-header';
+      templateValues.messageText =
+        'The link you clicked to confirm your email is expired.';
+      templateValues.messageId = 'signin-link-expired-message';
+
+      break;
+    default:
+      throw new Error('Invalid link type passed into LinkExpired component');
+  }
+  return templateValues;
+};
+
+const LinkExpired = ({ linkType }: LinkExpiredProps) => {
+  // TODO : Metric event(s) for expired link
+
+  const resendLinkHandler = getResendLinkHandler(linkType);
+  const templateValues = getTemplateValues(linkType);
+
+  return (
+    <>
+      {/* TODO: Add alertBar for success/failure status of resendLinkHandler */}
+      <FtlMsg id={templateValues.headerId}>
+        <h1 className="card-header">{templateValues.headerText}</h1>
+      </FtlMsg>
+
+      <FtlMsg id={templateValues.messageId}>
+        <p className="mt-4 text-sm">{templateValues.messageText}</p>
+      </FtlMsg>
+      <FtlMsg id="resend-link">
+        <button
+          onClick={resendLinkHandler}
+          className="cta-primary cta-base-p mt-4"
+        >
+          Receive new link
+        </button>
```

Comment:
I think this might be more consistent with our other pages to wrap `FtlMsg` in a div with `flex` and adding `cta-xl` but I think the original is just a simple link? Should we just use `link-blue` to match parity instead of making this a `cta-primary`?

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/13018240/213031631-3439f1f2-8eea-47a7-b46f-a0d91890d8e3.png">

Reply:
Oh sure, thanks for catching that!